### DISCUSSION TITLE
How to block applications from networking through the proxy...

### DISCUSSION THREAD

**Discussion by pomolio** [tnodir/fort#105] [Q&A] [answered]
I use v2ray program to proxy internet access
The blocked program will connect through 127.0.0.1
How to block it effectively....?

**Comment by tnodir** [comment, marked as answer]
- turn on the Options: Self Protection: "Filter Local Addresses" flag
- remove the "127.0.0.0/8" from Options: IP Addresses: Internet Addresses: "Exclude"

**Reply by pomolio** [reply]
Does this setting block all applications?
Can individual applications be blocked?

**Reply by tnodir** [reply]
1. The "Filter Local Addresses" flag disables auto-allowing of `127.0.0.1` and `255.255.255.255` addresses.
2. And after removing the `127.0.0.0/8` from Options: IP Addresses: Internet Addresses: "Exclude", the `127.0.0.1` address is not auto-allowing as LAN address.

So the `127.0.0.1` address treated as Internet address and blocked apps can not connect to it.
